### PR TITLE
Changelog for 2017.05.0 release

### PR DESCRIPTION
**Pull request by mkow**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+    * There's a possibility to script Veles using both Python 2 and 3, but currently without a support for any GUI operations.
+  * Created an instalator for Windows and added a support for DEB packages, which install all dependencies.
+  * Dropped 32-bit Windows support, due to problems with MessagePack.
+
+### Minor features and changes:
+
+  * Added more encoders in hex view ("Copy as" context menu).
+  * Added connection dialog.
+  * Lowered minimum macOS version to 10.10.
+  * Python3 is now needed to run Veles on macOS.
```
Comment (adamiwaniuk) [inline changelog.md]: python >= 3.5

File: changelog.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+  * On Ubuntu you need to enable "universe" repository (it is normally enabled by default, but not on live CD).
```
Comment (adamiwaniuk) [inline changelog.md]: to install veles from deb pacage

File: changelog.md [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
 
+## 2017.05.0.Świtezianka (2017-05-12)
+
+### Major features and changes:
+
+  * Introduced client-server architecture:
+    * The server can run remotely and can be shared amongst many users.
+    * The server is currently written in Python 3.
+    * There's a possibility to script Veles using both Python 2 and 3, but currently without a support for any GUI operations.
+  * Created an instalator for Windows and added a support for DEB packages, which install all dependencies.
```
Comment (adamiwaniuk) [inline changelog.md]: + drag-and-drop macOS X

File: changelog.md [hunk 4]
```diff
@@ -1,5 +1,29 @@
...
+  * Dropped 32-bit Windows support, due to problems with MessagePack.
+
```
Comment (adamiwaniuk) [inline changelog.md]: + save current state in database file

File: changelog.md [hunk 5]
```diff
@@ -1,5 +1,29 @@
...
+  * Added connection dialog.
+  * Lowered minimum macOS version to 10.10.
+  * Python3 is now needed to run Veles on macOS.
+  * On Ubuntu you need to enable "universe" repository (it is normally enabled by default, but not on live CD).
+
+### Bugfixes:
+
+  * Fixed a glitch in 3D view mouse manipulation.
+  * 3D views which are not currently visible don't use CPU/GPU resources anymore.
+
```
Comment (adamiwaniuk) [inline changelog.md]: improved stabitility

File: changelog.md [hunk 6]
```diff
@@ -1,5 +1,29 @@
...
+## 2017.05.0.Świtezianka (2017-05-12)
+
+### Major features and changes:
+
+  * Introduced client-server architecture:
+    * The server can run remotely and can be shared amongst many users.
+    * The server is currently written in Python 3.
+    * There's a possibility to script Veles using both Python 2 and 3, but currently without a support for any GUI operations.
+  * Created an instalator for Windows and added a support for DEB packages, which install all dependencies.
+  * Dropped 32-bit Windows support, due to problems with MessagePack.
```
Comment (adamiwaniuk) [inline changelog.md]: official

File: changelog.md [hunk 7]
```diff
@@ -16,13 +16,22 @@
...
 
   * Fixed a glitch in 3D view mouse manipulation.
   * 3D views which are not currently visible don't use CPU/GPU resources anymore.
+  * Fixed minor stability issues.
+
+### Caveats:
+
+  * Python3 is now needed to run Veles on macOS and has to be manually installed.
+  * On Ubuntu you need to enable "universe" repository (it is normally enabled by default, but not on live CD).
+  * Hotkeys don't work on Ubuntu 16.\* due to a Qt bug:
```
Comment (adamiwaniuk) [inline changelog.md]: only from top menu

File: changelog.md (hunk 3 above)
Comment (mkow) [inline changelog.md]: Fixed.

File: changelog.md (hunk 6 above)
Comment (mkow) [inline changelog.md]: Fixed.

File: changelog.md (hunk 4 above)
Comment (mkow) [inline changelog.md]: Added.

File: changelog.md (hunk 7 above)
Comment (mkow) [inline changelog.md]: Added.

**Pull request merged**